### ISSUE TITLE
Xmage Freezes With Recross the Paths and Goblin Charbelcher  

### ISSUE THREAD

**Issue by Pipediban** [magefree/mage#10148]
Hello, XMage crashes with cards like Goblin Charbelcher or Recross the Paths.

Cards that says "reveal cards until..." crashes the games. 

It doesn't say some error in the screen, but the screen freezes and can't press any button, like concede for example.

Please can you fix that?

**Comment by Pipediban** [comment]
#Bug

**Comment by Grath** [comment]
I can't replicate this on the latest beta, whether I have land in my deck (for either effect to 'reveal until') or not.

**Comment by Grath** [comment]
I can't reliably reproduce this, but I did get it to occur on the public Beta server with [[Recross the Paths]] (once out of five attempts), at which point both clients (I was playing against myself) froze up during the 'select an opponent to clash with' step and couldn't click anything to progress.

**Comment by Pipediban** [comment]
Yes, in the public Beta server happend. So the freeze is because of the
clash?

**Comment by Pipediban** [comment]
And with Goblin Charbelcher it freezes to?

**Comment by Grath** [comment]
Charbelcher never froze in my testing with two players on public Beta, out of ~six activations.

**Comment by Pipediban** [comment]
Oh perfect, thanks

**Comment by Pipediban** [comment]
Recross the Paths is still bugged. All game freezes.

[image: image.png]

**Comment by Pipediban** [comment]
Can you fix it please?

**Comment by Grath** [comment]
For now, your best bet is going to be playing other cards.

**Comment by Pipediban** [comment]
Ok, thanks.

**Comment by Susucre** [comment]
I suspect Recross the Path was among the broken clash cards fixed recently. [[Goblin Charbelcher]] is  another bug, not sure it is still broken.
